src/components/ui/Chat/ChannelList/ChannelItem.tsx

```diff
@@ -22,25 +22,80 @@ export interface ChannelItemProps
   isSelected?: boolean;
   /** Callback function when clicked */
   onClick: () => void;
-  /** Signifies of there are unread message in the channel. */
+  /** Signifies if there are unread message in the channel. */
   unread?: boolean;
   /** Content of the badge signaling unread messages, such as a badge displaying the number. */
   unreadBadgeLabel?: string;
+  /** Indicate if we want to display the detailed channel view with last channel message or timestamp */
+  displayDetailedChannelView?: boolean;
+  /* Content of the last channel message to be displayed */
+  lastChannelMessage?: string;
```

Comment:
From a builder perspective, these props do not need to be the *last* message or timestamp. i.e., the `ChannelItem` component does not check whether these props are really last messages or have any special message like `Last message: ${lastChannelMessage}`.

Rather, you can simply add one more prop, say `detailView`. If `detailView` exists, show it in a detailed view.

```
detailView: ReactNode // ReactNode is ReactElement, string, or any other element
```

```
detailView={
  <div className="ch-detailed-channel-message">
    {lastChannelMessage}
  </div>
  <div className="ch-detailed-channel-message-time">
    {lastChannelMessageTimestamp}
  </div>
}
```

Reply:
Actually, NVM.. I just reviewed screenshots again. You can't just pass down to show the date and time on the right side.